Real_world_examples/Water_quality_suspended_matter.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<p>It might be worth adding the following for some more background:</p><ul><li>The indicator is part of the Sustainable Development Goal 6 - "Ensure access to water and sanitation for all". Maybe we can add an image as well? <a href="https://www.monash.edu/msdi/assets/images/icons/E_SDG-goals_icons-individual-rgb-06.png" target="_blank">https://www.monash.edu/msdi/assets/images/icons/E_SDG-goals_icons-individual-rgb-06.png</a></li><li>What <em>is </em>suspended matter and how does it relate to the concept of turbidity?</li></ul><p> </p><p>I am not sure whether we want to diverge from the text on the DE Africa Notebook though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
That's a good idea - I have added in the indicator text. Re the image - I am not sure about licensing of image - maybe safer to keep it out